src/jdk.incubator.vector/share/classes/jdk/incubator/vector/ByteVector.java

```diff
@@ -3905,8 +3905,8 @@ final ByteVector fromIntValues(int[] values) {
         // Virtual constructors
 
         @ForceInline
-        @Override final
-        public ByteVector fromArray(Object a, int offset) {
+        @Override public
```

Comment:
`public` should probably go on a different line? (same as for the ones that follow)

Reply:
Note these files are generated from a template `X-Vector.java.template` and the script `gen-src.sh` (not yet integrated into the build system). So the template requires updating, from which the script is run to generate the source. We can do this after integration of https://github.com/openjdk/jdk/pull/5873/